jupyterhub/tests/test_services_auth.py

```diff
@@ -217,7 +219,8 @@ def finish_thread():
         assert auth.login_url in r.headers['Location']
 
 
-def test_service_cookie_auth(app, mockservice_url):
+def test_hubauth_cookie(app, mockservice_url):
```

Comment:
For the simple case here, it doesn't matter. For some of the longer tests, it may make sense to parameterize the input test case since I think the test becomes more readable and more useful later on (with multiple services and hubshare adding complexity). The test can remain the same even if there are additional test input needed. For now, I think they are fine as is.